code/game/objects/items/tanks/watertank.dm

```diff
@@ -147,7 +147,7 @@
 	desc = "A janitorial cleaner backpack with nozzle to clean blood and graffiti."
 	icon_state = "waterbackpackjani"
 	item_state = "waterbackpackjani"
-	custom_price = 100
+	custom_price = 1200
```

Comment:
This costs 1k from cargo. Stuff from vendors should probably be slightly cheaper than cargo since you don't get a crate to sell.

Reply:
Vendors should always have a 10% markup from cargo I would argue.

Reply:
at least 10% if not more, especially cause you require no interaction with others to get it.

Reply:
Some of the prices for cargo crates are pretty off from what I would consider reasonable (6k credits for a couple of pizzas to make an example) so I didn't use a specific markup rate, but yes cargo is generally the better option for the reasons stated, especially for equipment

Also worth noting that this is still free if you have the relevant access, departmental wardrobe vendors are similarly expensive, get access or cough up the cash

Reply:
I agree with oranges so I think this is resolved.